webpack.common.js

```diff
@@ -47,6 +47,11 @@ module.exports = {
           },
           {
             loader: "css-loader",
+            options: {
```

Comment:
Uten denne ble verdi på fargekonstantene `undefined`, fant løsningen her: https://nav-it.slack.com/archives/C7NE7A8UF/p1602845017068300